### PR TITLE
Conditional election updates to "How government works"

### PR DESCRIPTION
**Pull request by dsingleton**
Makes some of the content of [how government works](https://www.gov.uk/government/how-government-works) conditional on the government, so that when a new government forms the page does not show content specific to the 2010-2015 government, and is politically neutral. 

The condition is based on the current government having a `start_date` after polling day, so that once the new government is created the page will update automatically (subject to caching).

The first couple of commits are general bug/copy fixes, not specific to any government. The last commit introduces the conditional logic and content, with more detail on the specific changes.

This does introduce some tech debt, with all the conditionals and duplicate content, will be short lived and cleaned up by the election team, when the page is properly updated for the new government.

- [x] Product review from @bmwelby 
- [x] Code review

| Before | After |
|---|---|
| how-gov-works-before | how-gov-works-after |

### CODE REVIEW

**Comment by dsingleton** [comment]
(Updated the new images to be better compressed)

File: app/views/home/how_government_works.html.erb [hunk 1]
```diff
@@ -25,9 +47,9 @@
             <h3>The Prime Minister</h3>
             <p>The <a href="/government/ministers/prime-minister">Prime Minister</a> is head of the UK government. He is ultimately responsible for all policy and decisions. He:</p>
             <ul>
-              <li>oversees the operation of the Civil Service and government agencies</li>
-              <li>appoints members of the government</li>
-              <li>is the principal government figure in the House of Commons</li>
+              <li>oversee the operation of the Civil Service and government agencies</li>
```
Comment (evilstreak) [inline app/views/home/how_government_works.html.erb]: These change don't seem right. "He oversee the operation"

**Comment by tekin** [comment]
:+1:

File: app/views/home/how_government_works.html.erb (hunk 1 above)
Comment (dsingleton) [inline app/views/home/how_government_works.html.erb]: Yep, i messed that up when re-ordering @bmwelby's commits, will fix with a rebase.

**Pull request merged**